mlflow_faculty/trackingstore.py

```diff
@@ -258,6 +262,12 @@ def create_run(
                 None if parent_run_id is None else UUID(parent_run_id),
                 tags=[mlflow_tag_to_faculty_tag(tag) for tag in tags],
             )
+        except FacultyExperimentNotActiveConflict as conflict:
+            raise MlflowException(
+                "Non active experiment with id: {}".format(
```

Comment:
I think we decided to have the exception message contain an instruction on how to undelete an experiment.

The easiest way to do this is:

```
mlflow experiments restore <experimentId>
```

So we could have a message like:

```
Experiment <id> is deleted. To create runs for this experiment, first restore it with the shell command 'mlflow experiments restore <id>'.
```